style.css

```diff
@@ -480,3 +475,168 @@ footer {
   height: 4.5rem;
   border-top: solid 2px #dfe1e6;
 }
+
+.desktopMenu {
+  display: none;
+}
+
+@media (min-width: 768px) {
+  .mobileMenu {
+    display: none;
+  }
+
+  .desktopMenu {
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    gap: 2rem;
+    list-style: none;
+    padding-right: 3rem;
+  }
+
+  .desktopLink {
+    font-family: "Inter", serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  .mail {
+    width: 2.5rem;
+    height: 2.5rem;
+    padding-top: 0.25rem;
+  }
+
+  .bgImage {
+    position: absolute;
+    background-image: url("./images/Desk-Illustration-header.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+    width: 27rem;
+    right: 0;
+    top: 0%;
+    z-index: 20;
+  }
+
+  .headline {
+    flex-direction: row;
+  }
+
+  .headerText {
+    margin-left: 2rem;
+    height: 10px;
+  }
+
+  .headline > .socials {
+    flex-direction: column;
+    justify-content: center;
+    margin-right: auto;
+    order: -1;
+  }
+
+  .socialLink {
+    width: 1.5rem;
+    height: 1.5rem;
+  }
+
+  .headlineDesc {
+    width: 50%;
+  }
+
+  .recentWorks {
+    flex-direction: row;
+  }
+
+  .recentWorks > h2 {
+    white-space: nowrap;
+  }
+
+  .recentWorks > img {
+    margin-left: 1.5rem;
+    width: 100%;
+  }
+
+  .projects {
+    flex-direction: row;
+  }
+
+  .cards {
+    display: grid;
+    grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+    grid-gap: 1.5rem;
+  }
+
+  .cardBtns {
+    display: flex;
+    gap: 2%;
+  }
+
+  .cardContainer {
+    height: auto;
+    position: relative;
+  }
+
+  .cardSeeProj {
+    display: none;
+  }
+
+  .cardContainer:hover .cardSeeProj {
+    display: block;
+    position: absolute;
+    bottom: 0;
+  }
+
+  .cardContainer:hover .cardBtns {
+    display: none;
+  }
+
+  .sideTools {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    align-items: start;
+  }
+
+  .about,
+  .work {
+    padding: 1rem 4.875rem;
+  }
+
+  .aboutTools {
+    align-items: start;
+    justify-items: center;
+  }
+
+  .about > p {
+    width: 50%;
+  }
+
+  .about {
+    background-image: url("./images/illustration-about\ me.svg");
+    background-position: top 100px right;
+    background-size: 23rem 23rem;
+  }
+
+  .action {
+    margin-bottom: 2rem;
+  }
+
+  .contact {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+  }
+
+  .contactTitle {
+    text-align: start;
+    width: 80%;
+  }
+
+  .contactBtn {
+    align-self: flex-start;
+  }
+
+  .contactForm {
+    gap: 0;
+  }
+}
```

Comment:
**1-** After both the `input tags` you have used some images as separators. They are networking please consider reviewing them! to make it in accordance with the design.

**2-** I've reviewed your design and noticed that it doesn't quite match the Figma template. The main issue I see is that there are differences in the font sizes, widths, heights, margins, and paddings. Could you please take a closer look and adjust these elements to match the template more closely?


**I am attaching screenshots of both designs to make them more clear! 👍** 

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220551311-3f19b055-f9a5-4205-a0bc-ea0c96bf21af.png)

## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/220551287-0257164d-8100-4bf8-9a96-775f3ed252de.png)

## Remember:-
You did great work but the project demands to make it as close as possible to the design almost 100% exact. So your design font-sizes, margins, padding, width, and height are actually similar to the design.